x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/utils/get_finding_colors.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiThemeComputed } from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme'; // TODO: replace with euiTheme
+import type { VulnSeverity } from '@kbn/cloud-security-posture-common';
+import {
+  VULNERABILITIES_SEVERITY,
+  MISCONFIGURATION_STATUS,
+} from '@kbn/cloud-security-posture-common';
+
+const isBorealis = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('borealis');
+};
+
+const isAmsterdam = (euiThemeName: string) => {
+  return euiThemeName?.toLowerCase().includes('amsterdam');
+};
+
+// TODO: replace with severity color palette
+// TODO: replace with euiTheme.colors.vis.* from useEuiTheme hook
+// TODO: replace #aaa with proper color token
+export const getSeverityStatusColor = (
+  severity: VulnSeverity,
+  euiTheme: EuiThemeComputed
+): string => {
+  // TOOD: remove old mapping when severity palette is fixed (atm it is inverted)
+  if (euiTheme && isAmsterdam(euiTheme.themeName)) {
+    switch (severity) {
+      case VULNERABILITIES_SEVERITY.LOW:
+        return euiThemeVars.euiColorVis0;
+      case VULNERABILITIES_SEVERITY.MEDIUM:
+        return euiThemeVars.euiColorVis5_behindText;
+      case VULNERABILITIES_SEVERITY.HIGH:
+        return euiThemeVars.euiColorVis9_behindText;
+      case VULNERABILITIES_SEVERITY.CRITICAL:
+        return euiThemeVars.euiColorDanger;
+      default:
+        return '#aaa';
+    }
+  }
+
+  switch (severity) {
+    case VULNERABILITIES_SEVERITY.LOW:
+      return euiTheme.colors.vis.euiColorVis0; // TODO: use color from the severity palette?
```

Comment:
severity paletter introduced in https://github.com/elastic/eui/pull/8247 is missing green color which is used for low severity alerts and vulnerabilities. There is a discussion between EUI team and security designers on how to go about it. The workaround is to use `euiColorVis0` which is green in both themes. But if security designers agree to switch to blue info color for low severity signales, this line should be changed to `euiTheme.colors.vis.euiColorSeverity1`